### PR TITLE
Docusaurus 3.4 upgrade

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request upgrades repo to Docusaurus 3.4.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/platform-services/automation-service/app-central/integrations/imap.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ tags: [ cloud soar integrations ]
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/imap.png')} alt="imap" width="80"/>
+<img src={useBaseUrl('img/platform-services/automation-service/app-central/logos/imap.png')} alt="imap" width="80"/>
```
Comment (kimsauce) [inline docs/platform-services/automation-service/app-central/integrations/imap.md:9]:
```suggestion
<img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/imap.png')} alt="imap" width="80"/>
```

File: docs/platform-services/automation-service/app-central/integrations/imap.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ The final playbook looks like this:<br/><img src={useBaseUrl('/img/platform-serv
 
 * **IP**: ([0-9]+(?:\.[0-9]+){3})
 * **Email**: [\w\.-]+@[\w\.-]+
-* **URL**: (?i)(?:(?:https?|ftp):\/\/|www\.|ftp\.)(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[-A-Z0-9+&@#\/%=~\_|$?!:,.])\*(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[A-Z0-9+&@#\/%=~\_|$])
+* **URL**: `(?i)(?:(?:https?|ftp):\/\/|www\.|ftp\.)(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[-A-Z0-9+&@#\/%=~\_|$?!:,.])\*(?:\([-A-Z0-9+&@#\/%=~\_|$?!:,.]\*\)|[A-Z0-9+&@#\/%=~\_|$])`
```
Comment (kimsauce) [inline docs/platform-services/automation-service/app-central/integrations/imap.md:161]: Hi @jpipkin1 , with this upgrade, it's now stricter with some formatting. This regex query was breaking the Docs build, so I had to add backticks. Is this ok with you?

Comment (jpipkin1) [reply docs/platform-services/automation-service/app-central/integrations/imap.md:161]: Of course, @kimsauce! I didn't write those anyway. Whatever you have to do to make it work is okay

**Review by jpipkin1 — APPROVED** [review]

File: docs/platform-services/automation-service/app-central/integrations/imap.md (hunk 2 above)
Comment (kimsauce) [reply docs/platform-services/automation-service/app-central/integrations/imap.md:161]: oh ok thank you!

**Pull request merged**